wire-ios/Wire-iOS/Sources/UserInterface/Components/Controllers/FullscreenImageViewController.swift

```diff
@@ -748,10 +748,10 @@ extension FullscreenImageViewController: MessageActionResponder {
         let sourceView: UIView
 
         // iPad popover points to delete button of container is availible. The scrollView occupies most of the screen area and the popover is compressed.
-        if action == .delete,
+        if action == MessageAction.delete,
```

Comment:
Why do we need these changes?

Reply:
xcode was throwing error about unknown type. But it's not happening anymore